db/migrate/20180924195258_create_books.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      t.string :genero
```

Comment:
Please do not use spanish name for fields.
General rule: All the code should be write in english. The only place you can write in other language is in I18n files.

Tips: look at the api documentation WBooks to find good name for fields

Reply:
Yes, I noticed it on the book_testing branch and changed it there but I could change it here too if needed.